response_operations_ui/views/collection_exercise.py

```diff
@@ -8,21 +8,35 @@
 from response_operations_ui.common.mappers import convert_events_to_new_format, map_collection_exercise_state
 from response_operations_ui.controllers import collection_instrument_controllers, sample_controllers, \
     collection_exercise_controllers, survey_controllers
-from response_operations_ui.forms import EditCollectionExerciseDetailsForm, CreateCollectionExerciseDetailsForm
+from response_operations_ui.forms import EditCollectionExerciseDetailsForm, CreateCollectionExerciseDetailsForm, \
+    RemoveLoadedSample
 
 logger = wrap_logger(logging.getLogger(__name__))
 
 collection_exercise_bp = Blueprint('collection_exercise_bp', __name__,
                                    static_folder='static', template_folder='templates')
 
 
+def get_success_message(success_key):
+    return {
+        'sample_removed_success': "Sample removed",
+        'sample_loaded_success': "Sample successfully loaded"
+    }[success_key]
```

Comment:
This will throw a keyError if an incorrect success key is passed through the query string. Using `{}.get(success_key, None)` might be better?

Reply:
ah ok I just mimic'd something that was used in the survey.py file, i'll change that